uportal-war/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -305,7 +305,7 @@
             return false;
           });
           // Console for debugging.
-          console.log("menu", menu, "menuToggle", menuToggle);
+          //console.log("menu", menu, "menuToggle", menuToggle);
```

Comment:
I agree with @ChristianMurphy -- it's better to remove the line than to comment it out.

I took a quick look at the full file.  There are already some other lines nearby -- including more `console.log` statements -- that are commented out.  It would be great to clean those up as well. :)